packages/x-tree-view/src/SimpleTreeView/SimpleTreeView.tsx

```diff
@@ -174,6 +174,13 @@ SimpleTreeView.propTypes = {
    * If you don't provide this prop. It falls back to a randomly generated id.
    */
   id: PropTypes.string,
+  /**
+   * Used to determine if a given item is editable.
+   * @template R
+   * @param {R} item The item to check.
+   * @returns {boolean} `true` if the item should be editable.
+   */
+  isItemEditable: PropTypes.func,
```

Comment:
This should probably not leak on `isItemEditable`

And the `useTreeViewLabel` plugin should not even be bundled on this `SimpleTreeView` component.
But to do this, I need to remove the default plugin list and instead hard-code the list of plugins used by the `SimpleTreeView` component, I plan to do it soon :+1: